bgpd/bgp_evpn_mh.h

```diff
@@ -351,6 +351,8 @@ extern int bgp_evpn_remote_es_evi_add(struct bgp *bgp, struct bgpevpn *vpn,
 		const struct prefix_evpn *p);
 extern int bgp_evpn_remote_es_evi_del(struct bgp *bgp, struct bgpevpn *vpn,
 		const struct prefix_evpn *p);
+extern void bgp_evpn_vni_es_remote_cleanup(struct bgp *bgp,
```

Comment:
can we use evi intsead of vni here - I'm trying to make these things more generic ahead of the MPLS development.